std/testing/asserts.ts

```diff
@@ -28,10 +28,8 @@ export function _format(v: unknown): string {
       compact: false,
       iterableLimit: Infinity,
     }))
-    : String(v);
-  if (typeof v == "string") {
-    string = `"${string.replace(/(?=["\\])/g, "\\")}"`;
-  }
+    : `"${String(v).replace(/(?=["\\])/g, "\\")}"`;
```

Comment:
What is the intended purpose of this code? It may be the cause of the mismatch with my expectations in https://github.com/denoland/deno/issues/7545#issuecomment-694603355. (That is, an unexpected extra backslash.)

Reply:
Well, based on your reply in the issue, I realize the assertions are actually asserting what I expected, so I guess all is well. But I'm still not sure what this line of code is for, out of my own curiosity.

Reply:
Was already there as the only escape mechanism, does not apply when Deno.inspect is involved tho.
Its either Deno.inspect or string.replace.